### PR TITLE
Updating MoPub Adapter to version 5.4.0.0

### PR DESCRIPTION
**Pull request by spilaka**

### CODE REVIEW

**Comment by chauduyphanvu1** [comment]
@spilaka - FYI we have a pending PR we'd love to see merged for this release too: https://github.com/googleads/googleads-mobile-android-mediation/pull/98.

File: ThirdPartyAdapters/mopub/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # MoPub Adapter for Google Mobile Ads SDK for Android Changelog
 
+## 5.4.0.0
+- Verified compatibility with MoPub SDK 5.4.0.
```
Comment (mopubmediation) [inline ThirdPartyAdapters/mopub/CHANGELOG.md]: Could you update the changelog to add something like `Fix a native ad crash for publishers importing MoPub's non-native modules`?

**Comment by spilaka** [comment]
@chauduyphanvu1. Updated with new Change log.

**Comment by chauduyphanvu1** [comment]
> @chauduyphanvu1. Updated with new Change log.

Looks good - thanks. Feel free to merge!

**Pull request merged**